pkg/humio/client.go

```diff
@@ -176,8 +177,22 @@ func (c *Client) addAuthHeaders(req *http.Request) *http.Request {
 	return req
 }
 
-func (c *Client) SetAuthHeaders(headers map[string]string) {
+func (c *Client) SetAuthHeaders(headers map[string]string) error {
+	if c.OAuthPassThru {
+		authHeader := headers[backend.OAuthIdentityTokenHeaderName]
+		idTokenHeader := headers[backend.OAuthIdentityIDTokenHeaderName]
+		if authHeader != "" && idTokenHeader != "" {
```

Comment:
```suggestion
		if authHeader != "" || idTokenHeader != "" {
```

Would this be more the intention? I think we want to ensure that _both_ of these headers are set in order to enable OAuth pass-thru, right?